apps/native/app/src/screens/login/testing-login.tsx

```diff
@@ -90,27 +89,6 @@ export const TestingLoginScreen: NavigationFunctionComponent = ({
   const { openBrowser } = useBrowser()
   const intl = useIntl()
   const [isLoggingIn, setIsLoggingIn] = useState(false)
```

Comment:
Removed the onAuthRequestInitated since it is now longer available in the auth library.

```testing-login.tsx:101 `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method. Error Component Stack: ```